### PR TITLE
feat: build using golang version 1.17.13

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: Title
Ticket: MEN-5754

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ stages:
   - trigger
 
 variables:
-  GOLANG_VERSION: "1.17.6"
+  GOLANG_VERSION: "1.17.13"
```
Comment (merlin-northern) [inline .gitlab-ci.yml:10]: could you please verify that we test with the same version we build?

**Review by kacf — APPROVED** [review]

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Same as in [here](https://github.com/mendersoftware/mender/pull/1021#pullrequestreview-1072328054). Let's keep the discussion in one place.

File: .gitlab-ci.yml (hunk 1 above)
Comment (tranchitella) [reply .gitlab-ci.yml:10]: not relevant

**Comment by tranchitella** [comment]
> Same as in [here](https://github.com/mendersoftware/mender/pull/1021#pullrequestreview-1072328054). Let's keep the discussion in one place.

I removed the changelog entry.

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**